lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -67,6 +66,8 @@ module Constants
         service: 'ml.googleapis.com',
         resource_type: 'ml_job'
       }
+
+      LATENCY_REGEX = /^[ \t]*(\d+)(\.\d+)?[ \t]*s[ \t]*$/
```

Comment:
Instead of putting this into the `Constants` module, I'd suggest making this a class variable and moving it to `confgure`, alongside `@cloudfunctions_tag_regexp` and the like. Also, the naming convention in this file uses `regexp` instead of `regex`...